### PR TITLE
MIP-874 Basic debug node implmentation

### PR DESCRIPTION
**Pull request by SagarKarwande**
# Description
Description why this change is needed and what changes are required for it.
1. Why?
- User needed to debug live values in streams pipelines
2. Changes
- Adds basic debug node implementation.

## Fixes

[MIP-874](https://datakrew.atlassian.net/browse/MIP-874)

## Type of change

- [ ] Bug fix
- [ ] Feature Enhancement
- [x] New feature

## Breaking change?

- [x] Non-breaking change
- [ ] Breaking change

# How has this been tested?

- [x] Manually Tested using postman/client
- [ ] Written testcase

### CODE REVIEW

File: apps/acqdat_core/lib/acqdat_core/streams/streams.ex [hunk 1]
```diff
@@ -53,15 +53,16 @@ defmodule AcqdatCore.Streams do
...
-    payload = parse_payload(data, data["mode"])
-
-    event = %Event{originator: orignator, timestamps: nil, payload: payload}
-    state = {event, %{}}
-
     case Cache.get_rooted_graph(project.pipelines.id) do
       {:ok, {root_id, graph}} ->
+        orignator =
+          get_orignator(data, data["mode"])
+          |> Map.put(:pipeline_id, project.pipelines.id)
```
Comment (niteshcode4u) [inline apps/acqdat_core/lib/acqdat_core/streams/streams.ex:60]: please remove single piping
```suggestion
          data
          |> get_orignator(data["mode"])
          |> Map.put(:pipeline_id, project.pipelines.id)
```

File: apps/acqdat_api/lib/acqdat_api_web/channels/streams/streams_channel.ex [hunk 2]
```diff
@@ -0,0 +1,7 @@
+defmodule AcqdatApiWeb.Streams.StreamsChannel do
+  use Phoenix.Channel
+
+  def join("streams:debug_pip_" <> identifier, _params, socket) do
+    {:ok, %{message: "Connected to streams channel"}, socket}
```
Comment (BandanaPandey) [inline apps/acqdat_api/lib/acqdat_api_web/channels/streams/streams_channel.ex:5]: @SagarKarwande are we not doing any authentication here, when this `streams channel` is joined on `join` function ?

Comment (SagarKarwande) [reply apps/acqdat_api/lib/acqdat_api_web/channels/streams/streams_channel.ex:5]: @BandanaPandey Authentication of user happens in `AcqdatApiWeb.UserSocket`.
For now we don't have any auth channel wise.

File: apps/acqdat_core/lib/acqdat_core/streams/streams.ex (hunk 1 above)
Comment (SagarKarwande) [reply apps/acqdat_core/lib/acqdat_core/streams/streams.ex]: @niteshcode4u Done

File: apps/acqdat_api/lib/acqdat_api_web/channels/streams/streams_channel.ex (hunk 2 above)
Comment (BandanaPandey) [reply apps/acqdat_api/lib/acqdat_api_web/channels/streams/streams_channel.ex:5]: ok @SagarKarwande

**Pull request merged**